drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -4583,13 +4583,16 @@ void RasterizerStorageGLES2::_render_target_allocate(RenderTarget *rt) {
 	GLuint color_internal_format;
 	GLuint color_format;
 	GLuint color_type = GL_UNSIGNED_BYTE;
+	Image::Format image_format;
 
 	if (rt->flags[RasterizerStorage::RENDER_TARGET_TRANSPARENT]) {
-		color_internal_format = GL_RGBA;
+		color_internal_format = GL_RGBA8;
 		color_format = GL_RGBA;
+		image_format = Image::FORMAT_RGBA8;
 	} else {
-		color_internal_format = GL_RGB;
+		color_internal_format = GL_RGBA;
```

Comment:
Should this be GL_RGB8?

Reply:
yup, that is what you get when you make the last changes early in the morning while rushing to work :)